installing/install_config/customizations.adoc

```diff
@@ -5,91 +5,152 @@ include::modules/common-attributes.adoc[]
-the web console to set a logout redirect URL.
+|featuregate.config.openshift.io
+|Enables xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[FeatureGates]
+so that you can use Tech Preview features.
 
-|DNS
-|You cannot modify the DNS settings for your cluster. You can
-xref:../../networking/dns-operator.adoc#dns-operator[view the DNS Operator status].
+|image.config.openshift.io
+|Configures how specific xref:../../openshift_images/image-configuration.adoc#image-configuration[image registries] should be treated (allowed, disallowed, insecure, CA details).
 
-|FeatureGate
-|You can
-xref:../../nodes/clusters/nodes-cluster-enabling-features.adoc#nodes-cluster-enabling[enable a FeatureGate]
-so that you can use Tech Preview features.
+|ingress.config.openshift.io
+|Configuration details related to xref:../../networking/ingress-operator.adoc#nw-installation-ingress-config-asset_configuring-ingress[routing] such as the default domain for routes.
 
-|Image
-|You can configure
-xref:../../openshift_images/image-configuration.adoc#image-configuration[image]
-settings.
+|oauth.config.openshift.io
+|Configures identity providers and other behavior related to xref:../../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[internal OAuth server] flows.
 
-|Infrastructure
-|In {product-title} {product-version}, you cannot customize the Infrastructure resource.
+|project.config.openshift.io
+|Configures xref:../../applications/projects/configuring-project-creation.adoc#configuring-project-creation[how projects are created] including the project template.
 
-|Ingress
-|To enable external access to {product-title} cluster services, configure the
-xref:../../networking/ingress-operator.adoc#configuring-ingress[Ingress Operator].
+|proxy.config.openshift.io
+|Defines proxies to be used by components needing external network access.  Note: not all components currently consume this value.
 
-|Network
-|You cannot modify your cluster networking after installation. If you must
-customize your network, you
-xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[customize networking during installation].
+|scheduler.config.openshift.io
+|Configures xref:../../nodes/scheduling/nodes-scheduler-default.adoc#nodes-scheduler-default[scheduler] behavior such as policies and default nodeselectors.
 
-|OAuth
-|Configure the
-xref:../../authentication/configuring-internal-oauth.adoc#configuring-internal-oauth[internal OAuth server].
+|===
 
-|Project
-|Configure xref:../../applications/projects/configuring-project-creation.adoc#configuring-project-creation[how projects are created].
+[id="operator-configuration-resources_{context}"]
+== Operator configuration resources
+
+These configuration resources are cluster scoped singletons which control behavior of a specific component as
```

Comment:
s/cluster scoped/cluster-scoped
Is there a better word for _singletons_? It is not a term we use, likely to _techie_ or _jargony_

Reply:
The concept here is that there is a single instance of this resource type for the entire cluster.  Open to suggestions.

Reply:
I think `cluster-scoped instance` is OK.